pkg/models/transaction.go

```diff
@@ -0,0 +1,64 @@
+package models
+
+import (
+	"github.com/imchukwu/karygo_backend/pkg/config"
+	"gorm.io/gorm"
+)
+
+type Transaction struct {
+	gorm.Model
+	TravelerId      uint     `json:"userId"`
+	NumOfSenders    uint     `json:"num_of_senders"`
```

Comment:
There is no need for NumOfSenders, we can get this by simple count anytime we havve the list of senders information, storing it i beleive is not optimized

Reply:
Okay